### PR TITLE
moved bg out of extend

### PR DESCRIPTION
**Pull request by IlyaAdlan**

### CODE REVIEW

**Review by IlyaAdlan — COMMENTED** [review]
should be fixed

**Comment by T0nj3** [comment]
Who is on this conflict, så that we can merge this soon?

**Review by T0nj3 — APPROVED** [review]

**Review by T0nj3 — APPROVED** [review]

File: index.html
Comment (cnnrbrn) [inline index.html:1]: background property needs to go inside extend property, e,g.

{
  extend: [
    background: ...
  }
}

Comment (cnnrbrn) [reply index.html:1]: I will close this PR.

Once the Tailwind config is fixed, create a new PR.

**Comment by T0nj3** [comment]
@cgrotteland92 Could you resolve these conflicts? :)

**Comment by cgrotteland92** [comment]
Conflict should be resolved now. Looked like there was a new and an old version of the searchbar together.

**Pull request merged**